src/App.js

```diff
@@ -17,7 +17,30 @@ const styles = theme => ({
     paddingTop: theme.spacing.unit * 20,
   },
 });
-
+class Tile extends React.Component{
+  render(){
+    const tileStyle = {
+      width: '150px',
+      height: '150px',
+      backgroundColor:this.props.color,
+      borderRadius: '5px',
+      boxShadow: '0px 0px 15px 0px',
+      paddingTop:'65px',
+      color: this.props.color,
+      display:'inline-block',
+    };
+    const tileSpanStyle = {
+      color: 'white',
+      fontFamily:'Nunito,sans-serif',
+      fontWeight:700
+  };
+    return (
```

Comment:
You don't need to make this a `Class Component`. This could as easily be a simple `Functional Component` as well, since we don't need any of the underlying lifecycle methods in React Components.